LiskHQ/lisk-sdk
# Add seed reveal in the block header - Closes #4931

### What was the problem?

This PR resolves #4931 

### How was it solved?

- Add hash onion config to test app
- Save used hash onion infomation in the database
- Save registered hash onion and when overwritten, cleanup all the used hash onions
- Add seed reveal in the header

### How was it tested?

- Add tests in the forging module
- Check if the seed reveal is persisted
- Add test to check if the seedReveal bytes are in the signed bytes
